AssetManagementTestProject/TestCases/US308_DisableUserTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+
+namespace AssetManagementTestProject;
+
+[TestFixture]
+public class US308_DisableUserTest : NUnitWebTestSetup
+{
+
+    protected ManageUserPage? ManageUserPage;
```

Comment:
Move all properties of all test cases into NunitWebTestSetup: 
![image](https://user-images.githubusercontent.com/79431746/207043529-b8782fe3-9986-4dc2-be0b-0cf92d19c047.png)

Reply:
Done